shell/app-shell/elements/pop-menu.js

```diff
@@ -0,0 +1,80 @@
+// code
```

Comment:
Maybe this is a new element pattern, to place these here as reminder that these could be included, and this one happens to have none? Just looking to understand the thought.

Reply:
I was trying it out as a pattern, donno if it's useful. I'll erase the empty one.